server/routes/offenceCodeDecisions/offenceCodeDecisions.ts

```diff
@@ -80,4 +98,57 @@ export default class OffenceCodeRoutes {
       })
     )
   }
+
+  private async placeholderValues(adjudicationNumber: string, user: User) {
+    const draftAdjudication = await this.placeOnReportService.getDraftAdjudicationDetails(
+      Number(adjudicationNumber),
+      user
+    )
+    const [prisonerDetails, associatedPrisoner] = await Promise.all([
+      this.prisonerDetails(draftAdjudication.draftAdjudication.prisonerNumber, user),
+      this.prisonerDetails(draftAdjudication.draftAdjudication?.incidentRole?.associatedPrisonersNumber, user),
+    ])
+    return {
+      offenderFirstName: properCaseName(prisonerDetails?.firstName),
+      offenderLastName: properCaseName(prisonerDetails?.lastName),
+      assistedFirstName: properCaseName(associatedPrisoner?.firstName),
+      assistedLastName: properCaseName(associatedPrisoner?.lastName),
+    }
+  }
+
+  // If a member of staff or prisoner has been searched for then additional view data is displayed.
+  private async viewDataFromDecisionForm(form: DecisionForm, user: User) {
+    const userId = form?.selectedDecisionData?.userId // This could be a staff username or a prisoner number
+    if (form?.selectedDecisionData?.userId) {
+      switch (decisionTree.findById(form.selectedDecisionId).getType()) {
+        case DecisionType.OFFICER:
+        case DecisionType.STAFF: {
+          const decisionStaff = await this.userService.getStaffFromUsername(userId, user)
```

Comment:
depending on the question type we need to look up different view data for the selected person (prisoner or staff)